ruleset-style.md

```diff
@@ -69,7 +75,7 @@ Here is an example ruleset pre-style guidelines:
 
 ```
 <ruleset name="WHATWG.org">
-  <target host="whatwg.org" />
+  <target host='whatwg.org' />
```

Comment:
Why did you change the quotes here? It's inconsistent with the rest of the example. Actually, we may want to add an entry saying "Use double quotes rather than single quotes," just for consistent.